src/test/bundles/07-helm-overrides/values-file/values.yaml

```diff
@@ -0,0 +1,12 @@
+# podinfo.replicaCount: 3
```

Comment:
I think we want this uncommented so the test below shows `values` overwriting the `valuesFiles` value